aten/src/ATen/cuda/CUDASanitizerFallback.cpp

```diff
@@ -0,0 +1,24 @@
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <c10/core/impl/GPUTrace.h>
+#include <torch/library.h>
+#include <iostream>
+
+void CUDASanitizerFallback(const c10::OperatorHandle& op, c10::DispatchKeySet dispatch_keys, torch::jit::Stack* stack) {
+    const c10::FunctionSchema& schema = op.schema();
+
+    const std::size_t num_arguments = schema.arguments().size();
+    std::vector<c10::IValue> inputs = torch::jit::last(*stack, num_arguments).vec();
+
+    op.redispatchBoxed(dispatch_keys & c10::DispatchKeySet(c10::DispatchKeySet::FULL_AFTER, c10::DispatchKey::CUDASanitizer), stack);
```

Comment:
You probably don't want onky redispatch, because if there is an internal call to a dispatched operator, you will end up hitting the CSAN key again and end up with   a hit for a kernel that is internal to a kernel you already are tracing with CSAN. Better is to exclude the CSAN key before redispatching